### PR TITLE
Added another item to the list of information Insights Operator collects

### PR DESCRIPTION
**Pull request by mmuehlfeldRH**
Please include this enhancement in the 4.5 version of the docs.

@openshift/team-documentation

### CODE REVIEW

**Comment by martinkunc** [comment]
ACK, looks good to me. Thank you Marc !

File: modules/insights-operator-what-information-is-collected.adoc [hunk 1]
```diff
@@ -18,5 +18,6 @@ Primary information collected by the Insights Operator includes:
 * Information about nodes marked as `NotReady`
 * Events for all namespaces listed as "related objects" for Degraded operator
 * Anonymized certificate signing requests (CSR) and information about the validity of certificates
+* Configuration settings stored in `ConfigMap` objects in the `openshift-config` name space
```
Comment (ahardin-rh) [inline modules/insights-operator-what-information-is-collected.adoc:21]: s/name space/namespace

**Comment by ahardin-rh** [comment]
@mmuehlfeldRH Thanks for this! Just one minor comment from me. Once that update is complete and your commits are squashed into one commit, we'll be all set. Thanks!

File: modules/insights-operator-what-information-is-collected.adoc (hunk 1 above)
Comment (mmuehlfeldRH) [reply modules/insights-operator-what-information-is-collected.adoc:21]: I fixed the typo.

**Comment by ahardin-rh** [comment]
@mmuehlfeldRH Thank you!

This is a minor update, so no QE review required.

**Pull request merged**